libmattermost.c

```diff
@@ -1466,16 +1525,58 @@ mm_purple_blist_remove_chat(MattermostAccount *ma, const gchar *channel_id)
 	}
 }
 
+static gboolean
+mm_channel_is_hidden(MattermostAccount *ma, const gchar *id) 
+{
+	GList *prefs;
+
+	for(prefs=ma->user_prefs; prefs != NULL; prefs = g_list_next(prefs)) {
+		MattermostUserPref *pref = prefs->data;
+		if(purple_strequal(pref->name,id)) {
+			if(purple_strequal(pref->category,"direct_channel_show") || purple_strequal(pref->category,"group_channel_show")) {
+				if(purple_strequal(pref->value,"false")) {
+					return(TRUE);

```

Comment:
`return(TRUE);` is an odd syntax that I don't think I've seen before.  What C style is that?

Reply:
eeeee.. frankly saying not sure why it was there, re-styled now :-)